client/state/action-types.js

```diff
@@ -585,6 +585,9 @@ export const READER_UPDATE_NEW_POST_EMAIL_SUBSCRIPTION = 'READER_UPDATE_NEW_POST
 export const RECEIPT_FETCH = 'RECEIPT_FETCH';
 export const RECEIPT_FETCH_COMPLETED = 'RECEIPT_FETCH_COMPLETED';
 export const RECEIPT_FETCH_FAILED = 'RECEIPT_FETCH_FAILED';
+export const REWIND_STATUS_ERROR = 'REWIND_STATUS_ERROR';
```

Comment:
Yes, `REWIND_STATUS_UPDATE_ERROR` makes sense to me.

> If the server reports an error, will it return the error in a 2XX HTTP response?

This is a good question. There is an `error` field in the the 2XX response but the original PR #10862 ignores it. @dereksmart do we expect something useful in that field?

<img width="510" alt="screen shot 2017-06-06 at 13 52 11" src="https://cloud.githubusercontent.com/assets/7767559/26830058/7e9ebb66-4abf-11e7-8376-1eac50015433.png">